README.md

```diff
@@ -1,3 +1,17 @@
 # Pulsar Edit
 
-This is the website repo for Pulsar Edit. It contains our documenation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+This is the website repo for Pulsar Edit. It contains our documentation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+
+
+## Guide
+
+### Recommended Tools:
+- Toolbox: We recommend using this as a development tool. It helps with containerization of development.
+  - This can be setup in most distributions of linux using your package manager or with brew. Brew can also be used on Mac but if that's not ideal we recommend getting directly from the [official website](https://containertoolbx.org/).
+  - Yarn (don't set this up just yet, follow the guide below)
+  
+### Setup
+- Clone the repo! Put this where you want it and go ahead and change directories to this.
```

Comment:
```suggestion
- Clone the repo! You can do this by using the following command:
`git clone https://github.com/pulsar-edit/pulsar-edit.github.io.git pulsar-site`
This will create a folder called `pulsar-site` at the current location of your terminal
```